src/main/java/io/confluent/examples/streams/microservices/util/ProduceOrders.java

```diff
@@ -0,0 +1,51 @@
+package io.confluent.examples.streams.microservices;
+
+import java.util.Arrays;
+import java.util.Properties;
+
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import io.confluent.kafka.serializers.AbstractKafkaAvroSerDeConfig;
+import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerializer;
+import java.util.Collections;
+
+import io.confluent.examples.streams.utils.MonitoringInterceptorUtils;
+import io.confluent.examples.streams.microservices.domain.Schemas.Topics;
+import static io.confluent.examples.streams.microservices.domain.beans.OrderId.id;
+import static io.confluent.examples.streams.avro.microservices.OrderState.CREATED;
+import static io.confluent.examples.streams.avro.microservices.Product.UNDERPANTS;
+import static io.confluent.examples.streams.microservices.domain.Schemas.Topics.ORDERS;
+import io.confluent.examples.streams.avro.microservices.Order;
+
+public class ProduceOrders {
+
+    public static void main(final String[] args) throws Exception {
+
+        final SpecificAvroSerializer<Order> mySerializer = new SpecificAvroSerializer<>();
+        final boolean isKeySerde = false;
+        mySerializer.configure(
+            Collections.singletonMap(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, "http://localhost:8081"),
+            isKeySerde);
+
+        final Properties props = new Properties();
+        props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
+        props.put(ProducerConfig.ACKS_CONFIG, "all");
+        props.put(ProducerConfig.RETRIES_CONFIG, 0);
+        MonitoringInterceptorUtils.maybeConfigureInterceptorsProducer(props);
+        final KafkaProducer<String, Order> producer = new KafkaProducer<String, Order>(props, new StringSerializer(), mySerializer);
```

Comment:
use try-with-resources for `Producer` like you did in `AddInventory` here and below.

Reply:
Ack